src/main/java/me/moodcat/database/entities/Room.java

```diff
@@ -0,0 +1,28 @@
+package me.moodcat.database.entities;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@Data
+@Entity
+@Table(name = "room")
+@ToString(of = {
+        "id",
+})
+@EqualsAndHashCode(of = "id")
+public class Room {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false)
+    private int id;
+
```

Comment:
Yes, but I will leave that for #49 , this was just to get my @JoinColumn working in ChatMessage.